android/KMEA/app/src/main/java/com/tavultesoft/kmea/KMManager.java

```diff
@@ -206,11 +206,14 @@
   public static final String KMDefault_KeyboardName = "EuroLatin (SIL) Keyboard";
   public static final String KMDefault_LanguageName = "English";
   public static final String KMDefault_KeyboardFont = "DejaVuSans.ttf";
+  public static final String KMDefault_KeyboardVersion = "1.8.1";
```

Comment:
I can see this may be a maintenance issue for us in the future ... I wonder if we can dynamically download the latest version in the build, just as we do with FV Keyboards. This doesn't have to be fixed in this PR.

Reply:
Noted in #3085